docker-compose.yaml

```diff
@@ -50,7 +50,7 @@ services:
 
   fund-builder:
     image: ghcr.io/espressosystems/espresso-sequencer/bridge:main
-    command: bridge deposit
+    command: espresso-bridge deposit
```

Comment:
Thanks. The command is still be "bridge" in the docker image. I kept it like this so we don't have to update it everywhere. But yeah it's confusing. Maybe we should use `ENTRYPOINT` instead for our docker images so we don't have to specify the command. What do you think @Ancient123 ?